packages/monaco-editor-wrapper/src/wrapper.ts

```diff
@@ -188,41 +188,41 @@ export class MonacoEditorLanguageClientWrapper {
         return true;
     }
 
-    getMonacoEditorWrapper() {
```

Comment:
This in particular is a bit interesting, so what we're getting back is not a wrapper then? But an instance of the editor application?

Reply:
Yes, the name was wrong

Reply:
Fixed ⬇️